scripts/validate_docstrings.py

```diff
@@ -301,7 +301,10 @@ def pandas_validate(func_name: str):
             )
         examples_source_code = "".join(doc.examples_source_code)
         for wrong_import in ("numpy", "pandas"):
-            if f"import {wrong_import}" in examples_source_code:
+            if (
+                f"import {wrong_import}" in examples_source_code
+                and "import pandas_gbq" not in examples_source_code
```

Comment:
Once the import is removed this can probably be undone

Reply:
Yes thanks, doing it right now :)